packages/charts/src/components/ScatterChart/ScatterChart.tsx

```diff
@@ -231,6 +241,7 @@ const ScatterChart: FC<ScatterChartProps> = forwardRef((props: ScatterChartProps
           tick={<YAxisTicks config={yMeasure} />}
           width={yMeasure?.label ? yAxisWidth + 10 : yAxisWidth}
           margin={yMeasure?.label ? { left: 200 } : 0}
+          orientation={useIsRTL() === true ? 'right' : 'left'}
```

Comment:
please don't call hooks inside props. extract it to `const isRTL = useIsRTL()` somewhere before the return and just use the value here.